aws/sdk/sync-models.py

```diff
@@ -18,3 +18,5 @@
         print(f'warning: cannot find: {source}')
         continue
     shutil.copyfile(source, Path('aws-models') / model)
```

Comment:
side note: this script is demand generated in that it looks at the models we already have. We might want to print a warning that tries to discover any models that we missed in the models directory